Lib/test/test_call.py

```diff
@@ -169,6 +169,36 @@ def test_oldargs1_2_kw(self):
         msg = r"count\(\) takes no keyword arguments"
         self.assertRaisesRegex(TypeError, msg, [].count, x=2, y=2)
 
+    def test_argtype0(self):
```

Comment:
It is not correct place for these tests. It is a place for tests for incorrect number of arguments.

There are already tests for `TypeError` in `getattr()` and `hasattr()` in `test_builtin`. You only needs to make them more specific (check for error message).

Reply:
Thanks, I have just moved the tests as suggested.